app/assets/stylesheets/application/pages.scss

```diff
@@ -0,0 +1,12 @@
+#page {
+  margin-bottom: 40px;
+
+  .actions {
+    margin-bottom: 20px;
+
+    a, form {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line